case_studies/redshift_estimation/prior.py

```diff
@@ -110,3 +112,69 @@ def sample(self) -> RedshiftTileCatalog:
         }
 
         return RedshiftTileCatalog(self.tile_slen, catalog_params)
+
+
+class RedshiftCSVPrior(CatalogPrior):
+    """Prior distribution of objects in an astronomical image.
+
+    Inherits from CatalogPrior, adding redshift. Load a csvfile for flux and redshift.
+    """
+
+    def __init__(
+        self,
+        *args,
+        **kwargs,
+    ):
+        """Initializes CatalogPrior."""
+        super().__init__(*args, **kwargs)
+        self.array = pd.read_csv('/home/../data/scratch/songju/STAR.csv').to_numpy()[:,1:7]
```

Comment:
In the future, we may want to make this the file path to the csv an argument, but for now it's fine to leave it hard-coded.